src/app/ApplicationTemplate.Shared.Views/Helpers/DispatcherQueueExtensions.cs

```diff
@@ -0,0 +1,255 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// https://github.com/CommunityToolkit/WindowsCommunityToolkit/blob/main/License.md
+// See reference: https://github.com/CommunityToolkit/WindowsCommunityToolkit/blob/main/Microsoft.Toolkit.Uwp/Extensions/DispatcherQueueExtensions.cs
+
+using System;
+using System.Runtime.CompilerServices;
+using System.Threading.Tasks;
+using Windows.UI.Core;
+
+namespace Microsoft.UI.Dispatching;
+
+internal static class DispatcherQueueExtensions
+{
+	/// <summary>
+	/// Convert <see cref="CoreDispatcherPriority"/> to <see cref="DispatcherQueuePriority"/>
+	/// </summary>
+	private static DispatcherQueuePriority ConvertToDispatcherQueuePriority(CoreDispatcherPriority priority)
+	{
+		switch (priority)
+		{
+			case CoreDispatcherPriority.Low:
+				return DispatcherQueuePriority.Low;
+			case CoreDispatcherPriority.Normal:
+				return DispatcherQueuePriority.Normal;
+			case CoreDispatcherPriority.High:
+				return DispatcherQueuePriority.High;
+			default:
+				return DispatcherQueuePriority.Normal;
+		}
```

Comment:
why would we still need this here? Don't we want to move on from UWP stuff?

Reply:
It's to make it like the UWP implementation since the default one doesn't provide any asynchronous method. So I just copied what was done on the Chinook project, I could update it because I can see why we don't want to use `CoreDispatcherPriority`.

Reply:
@jeanplevesque

Reply:
I removed the old stuff and I'm using `DispatcherQueuePriority` instead which is the correct way.